common/pom.xml

```diff
@@ -24,6 +24,18 @@
 
     <modelVersion>4.0.0</modelVersion>
     <packaging>jar</packaging>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
```

Comment:
Is it necessary to force the compilation version to 8, which may cause some compatibility issues

Reply:
Updated. It seems like an automatic update from IDE and I forgot to change it back.